### PR TITLE
Retrofit description strings onto some early test cases.

### PR DESCRIPTION
**Pull request by toddjonker**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by zslayton — APPROVED** [review]
Nice 👌

File: conformance/local_symtab.ion [hunk 1]
```diff
@@ -2,70 +2,80 @@
...
+        (each (toplevel   '#$11')
+              (toplevel   '#$11' :: 1)
+              (toplevel { '#$11'  : 1 })
+              (signals "Invalid symbol address:")))
+  (then "with multi-item local symtab"
+        (toplevel $ion_symbol_table::{symbols:["a", "b"]})
+        (each (toplevel   '#$12')
+              (toplevel   '#$12' :: 1)
+              (toplevel { '#$12'  : 1 })
+              (signals "Invalid symbol addressx:"))))
```
Comment (toddjonker) [inline conformance/local_symtab.ion:49]:
```suggestion
              (signals "Invalid symbol address:"))))
```

File: conformance/local_symtab.ion [hunk 2]
```diff
@@ -2,70 +2,80 @@
...
-
-// Non-string and null (local) `symbols` entries act like $0
-(ion_1_0
-  (then (toplevel $ion_symbol_table::{symbols:[a, 1, true, ()]}
+(ion_1_0 "User symbols expand as expected"
+  (then "with one-item local symtab"
+        (each (toplevel $ion_symbol_table::{symbols:["a"]})
+              (toplevel             '#$3'::{  '#$7':["a"]})
+              (toplevel '#$10'::'#$10' '#$10')
+              (produces      a::a         a)))
```
Comment (popematt) [inline conformance/local_symtab.ion:25]: The `(toplevel '#$10'::'#$10' '#$10')` should be in a `then` clause, right?
```suggestion
              (then (toplevel '#$10'::'#$10' '#$10')
                    (produces      a::a         a))))
```

Comment (toddjonker) [reply conformance/local_symtab.ion]: Yes, thank you.

**Review by popematt — APPROVED** [review]

**Pull request merged**